src/core/config.c

```diff
@@ -0,0 +1,139 @@
+#include "config.h"
+#include "xdpw.h"
+
+#include <dictionary.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+#include <wordexp.h>
+#include "iniparser.h"
+
+#define SYSCONFDIR "/etc"
```

Comment:
We need to grab that from Meson's options.

Reply:
good idea